src/extensions/abcNotationExtension.js

```diff
@@ -0,0 +1,23 @@
+import abcjs from 'abcjs';
+import markdownItNotesSheet from './libs/markdownItNotesSheet';
+import extensionSvc from '../services/extensionSvc';
+
+let abc;
+
+extensionSvc.onGetOptions((options, properties) => {
+  options.abc = properties.extensions.abc.enabled;
+});
+
+extensionSvc.onInitConverter(2, (markdown, options) => {
+  if (options.abc) {
+    markdown.use(markdownItNotesSheet, (val) => {
+      abc = val;
+    });
+  }
+});
+
+extensionSvc.onSectionPreview(() => {
+  if (document.querySelector('#abcSheetPaper') != null && abc != null) {
+    abcjs.renderAbc('abcSheetPaper', abc, {});
+  }
+});
```

Comment:
No need to do all of this. No need to have a markdown-it plugin. Please have a look at the mermaid extension. You only need to do this:

```js
extensionSvc.onGetOptions((options, properties) => {
  options.abc = properties.extensions.abc.enabled;
});

extensionSvc.onSectionPreview((elt) => {
  elt.querySelectorAll('.prism.language-abc')
    .cl_each(diagramElt => render(diagramElt.parentNode));
});
```